src/tiled/tileanimationeditor.cpp

```diff
@@ -392,6 +410,26 @@ void TileAnimationEditor::framesEdited()
     mApplyingChanges = false;
 }
 
+void TileAnimationEditor::setFrameTime()
+{
+    QItemSelectionModel *selectionModel = mUi->frameList->selectionModel();
+    const QModelIndexList indexes = selectionModel->selectedIndexes();
+
+    if (indexes.isEmpty()) {
+        mFrameListModel->setDefaultFrameTime(mUi->frameTime->value());
```

Comment:
I think the default animation time should change immediately when you change the value of the `frameTime` widget. And of course, without affecting the existing frames.